httpcore/__init__.py

```diff
@@ -24,29 +24,31 @@
 from ._sync.http_proxy import SyncHTTPProxy
 
 __all__ = [
```

Comment:
The list has been sorted alphabetically, and added `ProtocolError` and `ProxyError` that were missing I guess due to a mistake.

Reply:
> added ProtocolError and ProxyError that were missing I guess due to a mistake.

I guess we could add an extra validation step in `test_exceptions.py` for this, as a follow up? :-) Or maybe even a more general test case for "verify that all exported members of `httpcore` as identified by `vars(httpcore)` are present in `__all__`". Maybe even check that the list is sorted… (Without validation the list will go unsorted again soon!)

Reply:
That's a great idea, I've added the test for both the order and make sure that everything is in `__all__`.